bindings/kitsune_runner/src/context.rs

```diff
@@ -5,15 +5,43 @@ use wind_tunnel_runner::prelude::UserValuesConstraint;
 /// Kitsune specific agent context values.
 #[derive(Debug, Default)]
 pub struct KitsuneAgentContext {
+    /// The chatter instance.
     pub(crate) chatter: Option<Arc<WtChatter>>,
+    /// The number of messages to be create per interval.
+    /// Defaults to 3.
+    pub(crate) num_messages: Option<u8>,
 }
 
 impl UserValuesConstraint for KitsuneAgentContext {}
 
 impl KitsuneAgentContext {
-    pub fn get_chatter(&self) -> Arc<WtChatter> {
+    /// Get chatter instance.
+    pub fn chatter(&self) -> Arc<WtChatter> {
         self.chatter.clone().expect(
             "chatter is not set, did you forget to call `create_chatter` in your agent setup?",
         )
     }
+
+    /// Get chatter id.
+    pub fn chatter_id(&self) -> String {
+        self.chatter
+            .clone()
+            .expect(
+                "chatter is not set, did you forget to call `create_chatter` in your agent setup?",
+            )
+            .id()
+            .to_string()
+    }
+
+    /// Get configured number of messages per interval.
+    pub fn number_of_messages(&mut self) -> u8 {
+        if self.num_messages.is_none() {
```

Comment:
The context is mutable, would this be clearer with a Once* type?

Reply:
'll be removed

Reply:
https://github.com/holochain/wind-tunnel/pull/153/commits/b9e7882a5918c439791e7a9ab156f65c35b10cf0